protocols/cosi/packets.go

```diff
@@ -11,6 +11,12 @@ import (
 
 // The main messages used by CoSi
 
+// see https://github.com/dedis/cothority/issues/260
+// 0 - no check at all
+// 1 - check only at root
+// 2 - check at each level of the tree
+var VerifyResponse = 1
```

Comment:
Yes there was the previous way of sending the parameter in the message. Both are strange but the "external api" issue is not yet discussed so .. it will change once we move onto that subject.